python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/resources.py

```diff
@@ -121,6 +128,34 @@ def _exec_sling_cmd(
             if proc.returncode != 0:
                 raise Exception("Sling command failed with error code %s", proc.returncode)
 
+    def stream(
+        self,
```

Comment:
Let's enforce these as kwargs, since there are a mix of Dagster + Sling specific arguments here.
```suggestion
    def stream(
        self,
        *,
```

Reply:
smart catch